FireRender.Maya.Src/FireRenderVolume.cpp

```diff
@@ -464,8 +464,25 @@ bool NorthstarRPRVolume::TranslateVolume()
 		auto albedoGridNode = frw::GridNode(context()->GetMaterialSystem());
 		albedoGridNode.SetGrid(m_albedoGrid);
 
-		auto albedoLookupNode = CreateLookupTextureNode(this, vdata.albedoGrid.valuesLookUpTable, albedoGridNode);
-		volumeShader.xSetValue(RPR_MATERIAL_INPUT_COLOR, albedoLookupNode);
+		float maxAlbedoValue = *std::max_element(vdata.albedoGrid.valuesLookUpTable.begin(), vdata.albedoGrid.valuesLookUpTable.end());
+
+		if (vdata.emissionGrid.IsValid() && maxAlbedoValue > 1)  //we need to normalize albedo with enabled emission
```

Comment:
Does albedo values > 1 produce correct color result when emission is disabled?

Reply:
Yes. Results are same with rpr1. But with emission this produces incorrect result.

Comment from Core unit tests:  // <- if using EMISSION, it's important to have COLOR lower than 1.0. Because EMISSION is weighted by (1.0 - color)

Reply:
OK